test/browser.js

```diff
@@ -316,13 +324,15 @@ test('headers are preserved when input is a Request and there are searchParams i
 		response.end();
 	});
 
+	await page.setBypassCSP(true);
```

Comment:
Almost didn't notice this. What's it for?

Reply:
Just forgot to remove it after testing something.